content/configuration/dev-server.md

```diff
@@ -208,6 +209,21 @@ To disable `contentBase`:
 contentBase: false
 ```
 
+### `devServer.watchContentBase`
+
+`boolean`
+
+Tell the server to watch the files served by the contentBase option. File changes will trigger a full page reload.
```

Comment:
Could you change "contentBase" to "`devServer.contentBase`" here (also note the backticks)?